email/src/main/java/edu/jhuapl/dorset/demos/EmailManager.java

```diff
@@ -0,0 +1,393 @@
+            }
+            errorFolder = store.getFolder(EmailType.ERROR.getValue());
+            completeFolder = store.getFolder(EmailType.COMPLETE.getValue());
+            inboxFolder = store.getFolder(EmailType.INBOX.getValue());
+        } catch (MessagingException e) {
+            logger.error("Failed to initialize folders");
+        }
+    }
+
+    /**
+     * Determine which folder should be accessed
+     * 
+     * @param folder   the folder to be returned. Expressed by an integer value
+     * @return Folder   the folder to be accessed
+     */
+    public Folder determineFolder(String folder) {
+        switch (EmailType.getType(folder)) {
+            case INBOX:
+                return inboxFolder;
+            case ERROR:
+                return errorFolder;
+            case COMPLETE:
+                return completeFolder;
+            default:
+                return null;
+        }
+    }
+
+    /**
+     * Determine which message array should be used
+     * 
+     * @param msg   the message to be accessed. Expressed by an integer value
+     * @return Message   the message to be used
+     */
+    public Message[] determineMsg(String msg) {
+        switch (EmailType.getType(msg)) {
+            case UNREAD:
+                return unreadMessages;
+            case PROCESSING:
+                return processingMessages;
+            default:
+                return null;
+        }
+    }
+
+    /**
+     * Open specified folder
+     * 
+     * @param folder   the folder to be opened. Expressed by an integer value  
+     */
+    public void openFolder(String folder) {
+        try {
+            determineFolder(folder).open(Folder.READ_WRITE);
+        } catch (MessagingException e) {
+            logger.error("Failed to open folders");
+        }
+    }
+
+    /**
+     * Return whether specified folder is open or not
+     * 
+     * @param folder  the folder to be tested. Expressed by an integer value
+     * @return boolean   whether the folder is open or not
+     */
+    public boolean folderIsOpen(String folder) {
+        return determineFolder(folder).isOpen();
+    }
+
+    /**
+     * Close specified folder
+     * 
+     * @param folder   the folder to be close. Expressed by an integer value
+     */
+    public void closeFolder(String folder) {
+        try {
+            determineFolder(folder).close(false);
+        } catch (MessagingException e) {
+            logger.error("Failed to close folders");
+        }
+    }
+    
+    /**
+     * Close store
+     */
+    public void closeStore() {
+        try {
+            store.close();
+        } catch (MessagingException e) {
+            logger.error("Failed to close store");
+        }
+    }
+
+    /**
+     * Return number of messages in specified folder
+     * 
+     * @param folder   the folder to look in. Expressed by an integer value
+     * @return int value   messages left in folder
+     */
+    public int getCount(String folder) {
```

Comment:
These methods would be much cleaner if you pass an instance of the Enum rather than a string. Then when it is called it looks like:

```java
emailsLeftInbox = email.getCount(EmailType.INBOX);
```